addons/web/static/src/fields/field_tooltip.js

```diff
@@ -0,0 +1,37 @@
+/** @odoo-module **/
+
+import fieldRegistry from "web.field_registry";
+
+export function tooltipInfo(params) {
+    let widgetDescription = undefined;
+    if (params.activeField.widget) {
+        if (params.activeField.FieldComponent.name === "LegacyField") {
+            const widget = fieldRegistry.get(params.activeField.widget);
+            widgetDescription = widget.prototype.description;
```

Comment:
is `widget.prototype.description` the same as `widget.constructor.description` ?

Reply:
it's the way to retrieve "static" properties on legacy classes